liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppsMojo.java

```diff
@@ -185,28 +204,33 @@ private boolean mavenWarPluginExists(MavenProject proj) {
         }
         return false;
     }
-    
+
     private boolean matches(Artifact artifact, ArtifactItem assemblyArtifact) {
         return artifact.getGroupId().equals(assemblyArtifact.getGroupId())
                 && artifact.getArtifactId().equals(assemblyArtifact.getArtifactId())
                 && artifact.getType().equals(assemblyArtifact.getType());
     }
-    
+
     private boolean isSupportedType(String type) {
         boolean supported = false;
         switch (type) {
-            case "ear":
-            case "war":
-            case "rar":
-            case "eba":
-            case "esa":
-            case "liberty-assembly":
+        case "ear":
+        case "war":
+        case "rar":
+        case "eba":
+        case "esa":
+        case "liberty-assembly":
+            supported = true;
+            break;
+        case "jar":
+            if (installThinProject) {
```

Comment:
For all the cases where 'thin' is not supported, I wonder if this method should catch that? So for example, under `case "ear":` should it check for `if (installThinProject)` and break without setting `supported = true;`?

Reply:
Done. Will throw an exception if the packaging type does not support thin-project configuration